integration/commands_administration_test.go

```diff
@@ -764,6 +764,111 @@ func TestCommandsAdministrationDBStatsEmptyWithScale(t *testing.T) {
 	// https://github.com/FerretDB/FerretDB/issues/727
 }
 
+func TestCommandsAdministrationRenameCollection(t *testing.T) {
+	t.Parallel()
+	insertCollections := []string{"foo", "none"}
+
+	for name, tc := range map[string]struct { //nolint:vet // for readability
+		collection       string
+		targetCollection string
+		expected         bson.D
+		err              *mongo.CommandError
+	}{
+		"Rename": {
+			collection:       "foo",
+			targetCollection: "bar",
+			expected:         bson.D{{"ok", float64(1)}},
+		},
+		"RenameSame": {
+			collection:       "foo",
+			targetCollection: "foo",
+			err: &mongo.CommandError{
+				Code:    20,
+				Name:    "IllegalOperation",
+				Message: `Can't rename a collection to itself`,
+			},
+		},
+		"RenameDuplicate": {
+			collection:       "foo",
+			targetCollection: "none",
+			err: &mongo.CommandError{
+				Code:    48,
+				Name:    "NamespaceExists",
+				Message: `target namespace exists`,
+			},
+		},
+		"RenameVerifyMetadata": {
+			collection:       "foo",
+			targetCollection: "bar",
+			expected:         bson.D{{"ok", float64(1)}},
+		},
+	} {
+		name, tc := name, tc
+		t.Run(name, func(t *testing.T) {
+			t.Parallel()
+
+			ctx, collection := setup.Setup(t, shareddata.Bools)
+
+			db := collection.Database()
+			require.NoError(t, db.Drop(ctx))
```

Comment:
Why do we drop here? Can't we just used `shareddata.Bools`?

Reply:
We shouldn't. I will fix.